mappings/net/minecraft/client/render/entity/LivingEntityRenderState.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/unmapped/C_ozphnurq net/minecraft/client/render/entity/LivingEntityRenderState
+	FIELD f_axhqulem pose Lnet/minecraft/unmapped/C_ufdjspmk;
+	FIELD f_cyjcipbw customName Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_fovngtjq skullType Lnet/minecraft/unmapped/C_hxsyxqpk$C_nyiksgwh;
+	FIELD f_hdaqnjbp headBodyYawDiff F
+	FIELD f_ihpohsqn isHurt Z
+	FIELD f_immbtgmx invisibleToPlayer Z
+	FIELD f_kcknwmag wornHeadAnimationProgress F
```

Comment:
```suggestion
	FIELD f_kcknwmag headFeatureAnimationProgress F
```
maybe? the name is a little confusing, i'm not sure if this is better

Reply:
It's specifically for when they're wearing an `AbstractSkullBlock` on their head, so I think it should include `worn`.
Though maybe `head` -> `skull`?

I kind of want to rename `AbstractSkullBlock` -> `AbstractHeadBlock`, but its block entity uses the string `"skull"` : /

Reply:
i think this name is a fine compromise, i don't think skull clarifies much